cmd/datalayer/showmymirrors.go

```diff
@@ -0,0 +1,97 @@
+package datalayer
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/chia-network/go-chia-libs/pkg/rpc"
+	"github.com/chia-network/go-chia-libs/pkg/types"
+	"github.com/chia-network/go-modules/pkg/slogs"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// showMyMirrorsCmd shows all mirrors owned by the user across all or specific stores
+var showMyMirrorsCmd = &cobra.Command{
+	Use:   "show-my-mirrors",
+	Short: "Shows all mirrors owned by the user across all or specific stores",
+	Example: `chia-tools data show-my-mirrors
+chia-tools data show-my-mirrors --id abcd1234`,
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		subID := viper.GetString("show-mirrors-id")
+		if subID != "" {
+			return nil
+		}
+		return nil
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		client, err := rpc.NewClient(rpc.ConnectionModeHTTP, rpc.WithAutoConfig())
+		if err != nil {
+			slogs.Logr.Fatal("error creating chia RPC client", "error", err)
+		}
+
+		subID := viper.GetString("show-mirrors-id")
+		if subID != "" {
+			// Show mirrors for specific store
+			ShowMirrorsForStore(client, subID)
+			return
+		}
+
+		// Show mirrors for all stores
+		subscriptions, _, err := client.DataLayerService.GetSubscriptions(&rpc.DatalayerGetSubscriptionsOptions{})
+		if err != nil {
+			slogs.Logr.Fatal("error getting list of datalayer subscriptions", "error", err)
+		}
+
+		for _, subscription := range subscriptions.StoreIDs {
+			ShowMirrorsForStore(client, subscription)
```

Comment:
The way this is written, it'll probably spit out a bunch of json strings like:
```
{"store_id": "hello", "mirrors": ["world"]}
{"store_id": "foo", "mirrors": ["bar","baz"]}
```
This may be fine for your intended usage. But if `ShowMirrorsForStore` was written to handle multiple subscriptions, it could just output something like:
```
{
   "subscriptions":[
      {
         "store_id":"foo",
         "mirrors":[
            "bar",
            "baz"
         ]
      },
      {
         "store_id":"hello",
         "mirrors":[
            "world"
         ]
      }
   ]
}
```
Which could be easier to consume in some situations with jq or similar CLI tools.

Reply:
This has been addressed